packages/react-ui/src/widgets/legend/LegendWrapper.js

```diff
@@ -33,9 +33,22 @@ const useStyles = makeStyles((theme) => ({
   }
 }));
 
+function UnknownLegendOption({ optionsKey, layerId }) {
+  return (
+    <p>
+      Unknown configuration key{' '}
+      <em>
+        <strong>{optionsKey}</strong>
+      </em>
+    </p>
+  );
+}
+
```

Comment:
I'd be more specific here and normalise the names.
optionsKey -> optionKey
UnknownLegendOption -> UnknownLayerOption
Unknown configuration key  -> Unknown layer option

Reply:
Ok, let's do it